azext_iot/tests/central/test_iot_central_devices_int.py

```diff
@@ -183,6 +183,19 @@ def test_central_device_methods_CRD(self):
             )
             is not None
         )
+
+        # UPDATE
+        new_device_name = f"{device_name}_new"
+        command = "iot central device update --app-id {} -d {} --device-name {}".format(
```

Comment:
Make the update tests here change all of the allowed updates (ex: enabling/disabling a device) and make sure the same is true for the other update tests.

Reply:
device PATCH has an ongoing bug on backend API. keep as is for now.
strengthen other tests